### PR TITLE
Add documentation for Bbox Sensors

### PR DESCRIPTION
**Pull request by HydrelioxGitHub**
**Description:**

Documentation for Bbox Sensors

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#3956

### CODE REVIEW

File: source/_components/sensor.bbox.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+logo: bbox.png
+ha_category: Sensor
+ha_release: 0.31
+---
+
+The `bbox` platform uses the [Bbox Modem Router](https://fr.wikipedia.org/wiki/Bbox/) from the French Internet provider Bouygues Telecom.
+Sensors are mainly bandwidth measures (Details below)
+
+<p class='note warning'>
+Due to third party limitation, the sensors will only be available if the HomeAssistant and the Bbox are on the same local area network.
```
Comment (Landrash) [inline source/_components/sensor.bbox.markdown]: HomeAssistant  -> Home Assistant .

File: source/_components/sensor.bbox.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+You can check this by going on the 192.168.1.254 address with your internet browser
```
Comment (Landrash) [inline source/_components/sensor.bbox.markdown]: I would repleace internet browser with Web browser. Also missing a ".".

File: source/_components/sensor.bbox.markdown (hunk 1 above)
Comment (Landrash) [inline source/_components/sensor.bbox.markdown]: HomeAssistant  -> Home Assistant .

File: source/_components/sensor.bbox.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    - up_max_bandwidth
+    - current_down_bandwidth
+    - current_up_bandwidth
+```
+
+Configuration variables:
+
+- **monitored_variables** array (*Required*): Sensors to display in the frontend.
+  - **down_max_bandwidth**: Maximum bandwidth you can use for downloading.
+  - **up_max_bandwidth**: Maximum bandwidth you can use for uploadinf.
```
Comment (Landrash) [inline source/_components/sensor.bbox.markdown]: uploadinf -> uploading.

File: source/_components/sensor.bbox.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    - down_max_bandwidth
+    - up_max_bandwidth
+    - current_down_bandwidth
+    - current_up_bandwidth
+```
+
+Configuration variables:
+
+- **monitored_variables** array (*Required*): Sensors to display in the frontend.
+  - **down_max_bandwidth**: Maximum bandwidth you can use for downloading.
```
Comment (Landrash) [inline source/_components/sensor.bbox.markdown]: I would probably rephrase this from 
>  Maximum bandwidth you can use for downloading.

To 
> Maximum bandwidth available for download.

File: source/_components/sensor.bbox.markdown (hunk 3 above)
Comment (Landrash) [inline source/_components/sensor.bbox.markdown]: I would probably rephrase this from 
>  Maximum bandwidth you can use for uploading.

To 
> Maximum bandwidth available for upload.

**Comment by fabaff** [comment]
Will fix the comments by @Landrash after merging.

**Pull request merged**